promql/promqltest/test.go

```diff
@@ -264,6 +267,40 @@ func parseSeries(defLine string, line int) (labels.Labels, []parser.SequenceValu
 	return metric, vals, nil
 }
 
+func parseExpect(defLine string) (expectCmdType, expectCmd, error) {
+	expectParts := patExpect.FindStringSubmatch(strings.TrimSpace(defLine))
+	expCmd := expectCmd{}
+	if expectParts == nil {
+		return 0, expCmd, fmt.Errorf("invalid expect statement, must match `%s`", patExpect.String())
```

Comment:
[nit] Including the regex in the error here might not be very clear - what if we included something like the syntax you've used in the readme?

Reply:
Yeah we can do that. I'll change this.